store/forms.py

```diff
@@ -14,4 +13,9 @@ class ProductAdminForm(forms.ModelForm):
     class Meta:
         model = Product
         exclude = ["store"]
-    
\ No newline at end of file
+
+
+class OrderAdminForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        exclude = ["store", "products"]
```

Comment:
should "total_cost" also be excluded?

Reply:
I like this. I didn't exclude it from the form but I did edit the widget to make the form control read-only.